controllers/platformnetwork_controller.go

```diff
@@ -883,7 +881,6 @@ func (r *PlatformNetworkReconciler) GetHostInstance(hostname string, request_nam
 func (r *PlatformNetworkReconciler) HandleHostStrategyUpdates(host_strategy cloudManager.HostStrategyInfo, request_namespace string) (err error) {
 	host_instance, err := r.GetHostInstance(host_strategy.Host.Hostname, request_namespace)
 	if err != nil {
-		logPlatformNetwork.Error(err, "Failed to get resource for host namespace %v", request_namespace)
```

Comment:
is the log message removed?

Reply:
yes, I noticed we are already logging it on `GetHostInstance`, hence removed it from other places.